sourcemash/api/feeds.py

```diff
@@ -104,8 +104,8 @@ def post(self):
                 q = Queue('scrape', connection=REDIS_CONNECTION)
```

Comment:
Is there a method on Connection to see if connection is valid/made? Then we can make this an if statement and resume 100%.